src/sql/azdata.d.ts

```diff
@@ -756,9 +756,7 @@ declare module 'azdata' {
 		flavor: string;
 	}
 
-	export interface QueryExecutionOptions {
-		options: Map<string, any>;
-	}
+	export type QueryExecutionOptions = { [option: string]: any };
```

Comment:
Should this be an interface?

Reply:
Interfaces and types are basically the same thing except interfaces have extends and some extra capabilities we don't need here. Essentially they're interchangeable.

Reply:
It might be better to be consistent with interfaces related to other types of requests. We use type for mostly different types of input (like enums) if you look around in the file.